src/containers/EditSubjectFrontpage/components/SubjectpageBanner.tsx

```diff
@@ -0,0 +1,121 @@
+/**
+ * Copyright (c) 2020-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree. *
+ */
+import React, { FC, Fragment, useEffect, useState } from 'react';
+import { injectT } from '@ndla/i18n';
+import { FieldHeader } from '@ndla/forms';
+import Modal, { ModalHeader, ModalBody, ModalCloseButton } from '@ndla/modal';
+import { FieldProps, FormikHelpers, FormikValues } from 'formik';
+import Button from '@ndla/button';
+import { TranslateType, VisualElement } from '../../../interfaces';
+import * as api from '../../../modules/image/imageApi';
+import { Image } from '../../../interfaces';
+import VisualElementSearch from '../../VisualElement/VisualElementSearch';
+import SubjectpageBannerImage from './SubjectpageBannerImage';
+
+interface Props {
+  t: TranslateType;
+  field: FieldProps<Image>['field'];
+  form: {
+    setFieldTouched: FormikHelpers<FormikValues>['setFieldTouched'];
+  };
+  bannerId: string;
+  locale: string;
+  isSavingImage: boolean;
+  title: string;
+}
+
+const SubjectpageBanner: FC<Props> = ({
+  t,
+  field,
+  form,
+  bannerId,
+  locale,
+  title,
+}) => {
+  const [showImageSelect, setShowImageSelect] = useState(false);
+  const [image, setImage] = useState<Image>();
```

Comment:
hva blir image når du har satt useState() til å være tom?

Reply:
Den blir vel undefined, og det skjer ved opprettelse av nye fagforsider. For ellers skal useEffect sørge for å fylle den når den rendrer...